src/ResourceManager/Sql/Commands.Sql/Data Sync/Cmdlet/NewAzureSqlSyncGroup.cs

```diff
@@ -0,0 +1,183 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Linq;
+using Microsoft.Azure.Commands.Sql.DataSync.Model;
+using Microsoft.Azure.Management.Sql.Models;
+using Hyak.Common;
+using Newtonsoft.Json.Linq;
+using System.IO;
+
+namespace Microsoft.Azure.Commands.Sql.DataSync.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create a new sync group
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmSqlSyncGroup", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.Low)]
+    public class NewAzureSqlSyncGroup : AzureSqlSyncGroupCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the sync group name
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The sync group name.")]
+        [ValidateNotNullOrEmpty]
+        public string SyncGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the frequency (in seconds) of doing data synchronization.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The frequency (in seconds) of doing data synchronization. Default is -1, which means the auto synchronization is not enabled.")]
+        public int? IntervalInSeconds { get; set; }
```

Comment:
Nice, perhaps consider adding IntervalInMinutes, IntervalInHours as different parameter sets so that user doesn't need to do math